presto-main/src/test/java/io/prestosql/sql/planner/TestLogicalPlanner.java

```diff
@@ -801,4 +806,11 @@ public void testDistributedSort()
                                                 tableScan("orders", ImmutableMap.of(
                                                         "ORDERKEY", "orderkey")))))));
     }
+
+    @Test
+    public void testDistinctOverInPredicate()
+    {
+        String sql = "SELECT custkey FROM orders WHERE custkey IN (SELECT distinct custkey FROM customer)";
```

Comment:
inline `sql`

Was this test failing before your change?

Reply:
This wasn't failing but it created the `Distinct` operation which is unnecessary.